components/ledger/cmd/app/main.go

```
@@ -37,11 +43,12 @@ func main() {
 	})
 	if err != nil {
-		logger.Errorf("Failed to initialize ledger service: %v", err)
-		_ = logger.Sync()
+		logger.Log(context.Background(), libLog.LevelError, fmt.Sprintf("Failed to initialize ledger service: %v", err))
+		_ = logger.Sync(context.Background())
 
 		os.Exit(1)
 	}
 
 	service.Run()
+	_ = logger.Sync(context.Background())
```

golangci-lint (2.11.3)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)